.firebaserc

```diff
@@ -0,0 +1,5 @@
+{
+  "projects": {
+    "default": "scrumlr-dev-530d9"
```

Comment:
Maybe we should add a .firebaserc with a dummy project name like `<YOUR_FIREBASE_PROJECT>` and add this file to .gitignore, after it was initially added?

Reply:
I'v don it.